### DISCUSSION TITLE
Can't seem to use the same markdown as the real ReadtheDocs markdown

### DISCUSSION THREAD

**Discussion by infracritical** [mkdocs/mkdocs#3150] [Q&A]
I've tried the notes one (.. NOTE::) - no such luck. What will work?

Many of the markdown functions don't seem to work with the 'readthedocs' theme.

Am I doing something?

**Comment by pawamoy** [comment]
`.. NOTE::` is not Markdown, it's reStructuredText. See https://www.sphinx-doc.org/en/master/usage/restructuredtext/index.html.
None of these RST directives (`.. thing::`) will work with MkDocs.

**Reply by infracritical** [reply]
Awwwwwww...nuts.  So do I have to get my own CSS file and include that as my special HTML tags?

**Reply by pawamoy** [reply]
RST and Markdown are indeed rendered to HTML (which you can customize with CSS), but through another language, in this case Python. You won't be able to interpret RST directives with just CSS, if that's what you meant.

If you want to use MkDocs, which entirely relies on Markdown, I would suggest you stop using RST and convert your documents to Markdown. Pandoc can help you converting from RST to Markdown.

**Reply by infracritical** [reply]
Yeah...no.  Using this is still much, much easier for documentation.

How can I create indented navigations within the sidebar?

I did this:

nav:
     - Home: home.md
          - Subhome: subhome.md
 
And yet...

it shows

Home
Subhome

It's not indented, and doesn't seem to follow similar rules of Markdown, let alone Sphinx.  Am I doing something wrong for the YML file?

**Reply by Andre601** [reply]
```yaml
nav:
- Home:
  - Home: index.md
  - Subhome: subhome.md
```

**Reply by pawamoy** [reply]
> Yeah...no. Using this is still much, much easier for documentation.

Not sure what you mean. If you mean to keep using RST, then MkDocs is the wrong tool for you :/

**Comment by Andre601** [comment]
If you mean admonitions...

```yaml
!!! note "title"
    Text
```

You need to keep an empty line before and after this text.
